packages/chakra-components/src/MenuBar/styles.js

```diff
@@ -6,7 +6,6 @@ import { useColorMode } from "@chakra-ui/core";
 
 export const useMenuBarStyle = () => {
   const baseProps = {
-    alignItems: "center",
```

Comment:
Please don't change the base styles for a prop change. 
Base styles are touched only if we decide to change the minimal styling.
Include this back